### DISCUSSION TITLE
How to install it for Ventura now? Tutorial is old?

### DISCUSSION THREAD

**Discussion by bigwowik** [tunglamvghy/AsusS530UN-hackintosh#46] [General]
I tried to instal by your tutorial, but I can't make a recovery USB with gibMacOS and MacOS 13, because I don't give a chance to downoad recovery for this version.
Can you help, how to install it for now on Asus Vivobook 15.
Please! Can you update tutorial or give step by step recomentations about it?
Very want to try it! Thank you for your work!

**Comment by tunglamvghy** [comment]
> I tried to instal by your tutorial, but I can't make a recovery USB with gibMacOS and MacOS 13, because I don't give a chance to downoad recovery for this version. Can you help, how to install it for now on Asus Vivobook 15. Please! Can you update tutorial or give step by step recomentations about it? Very want to try it! Thank you for your work!

Download the Dortania image to make an installation flash drive. Then replace the EFI folder with this repo.

**Reply by bigwowik** [reply]
Thanks! I will try it!
